data/models/dishes-model.js

```diff
@@ -0,0 +1,22 @@
+const db = require('../dbConfig');
+
+const getDishes = () => {
+  return db('dishes');
+};
+
+const addDish = dish => {
+  return db('dishes').insert(dish);
+};
+
+const getDish = id => {
+  return db('dishes')
+    .leftJoin('recipes', 'dishes.id', 'recipes.dishes_id')
+    .select('dishes.id', 'dishes.name as dish', 'recipes.name as recipe')
+    .where({ 'dishes.id': id });
```

Comment:
interesting, this exact same request will now return an array of recipes per single dish object as I mentioned would be preferred in your last  branch, but the code here is the same--did you change something in your seeds or migrations in between these two PRs?

Reply:
You may have already figured this out, but I use JS in the endpoint to take the SELECT data and turn it into the form the specifications asked for.